src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,71 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  addMovies: (movie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovies }) => {
+  const [searchTitle, setSearchTitle] = useState('');
+  const [movie, setMovie] = useState<Movie>();
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const changeMovie = async (inputValue: string) => {
+    setIsLoading(true);
+
+    const newMovie = await getMovie(inputValue)
+      .finally(() => setIsLoading(false));
+
+    if ('Error' in newMovie) {
+      setIsError(true);
+    } else {
+      setIsError(false);
```

Comment:
You can use then and catch methods instead

Reply:
I can't understand how to do it correctly

Reply:
Smth like this:
```
try {
  const newMovie = await getMovie(inputValue);
} catch(error) {
  setIsError(true);
} finally {
  setIsLoading(false);
}
```

But if you want to handle `Error` on API response, you can leave this as is